### DISCUSSION TITLE
to disable any resource attribute, we use environment variable OTEL_EXPERIMENTAL_RESOURCE_DISABLED_KEYS

### DISCUSSION THREAD

**Discussion by krishnaexplore** [open-telemetry/opentelemetry-java-instrumentation#10978] [Q&A] [answered]
to disable any resource attribute, we use environment variable OTEL_EXPERIMENTAL_RESOURCE_DISABLED_KEYS

is this variable name going to change in near future? what is convention on EXPERIMENTAL variables/features?

**Comment by laurit** [comment, marked as answer]
This property is defined in [opentelemetry-java](https://github.com/open-telemetry/opentelemetry-java) in https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#opentelemetry-resource so that question is better asked in that project. My guess would be that a configuration option is named `EXPERIMENTAL` if it is not defined in the open telemetry specification, once it is defined in the spec the experimental option will be replaced with what is in the spec.
